dask/dataframe/core.py

```diff
@@ -792,6 +792,15 @@ def map_partitions(self, func, *args, **kwargs):
         >>> res.dtype
         dtype('float64')
 
+        Here we apply a function to a Series resulting in a Series:
+
+        >>> d = [1, 2, 3, 4, 5]
+        >>> s = pd.Series(data=d)
+        >>> ds = dd.from_pandas(s, npartitions=2)
```

Comment:
Since this is a part of the border example, I'm wondering if it'll be nicer to re-use the same `ddf` in the example instead of defining a new Series?

Something like:

```python
 >>> res = ddf.x.map_partitions(lambda x: len(x)) # ddf.x is a Dask Series Structure
>>> res.dtype
dtype('int64')
```

Reply:
This makes a lot of sense for the suggestion! Added in commit [9595b03](https://github.com/dask/dask/pull/9161/commits/9595b03893f3976fb42dafa44195b12c6f3d99f5).